docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -122,9 +122,37 @@ what should be returned as well as implementing it.  This will
 certainly affect the syncer between mailbox workspace and edge
 cluster, and the summarization part will affect the status summarizer.
 
+### Good handling of workload conflicts
+
+We could start by handling workload conflicts in a very simple way:
+treating each as an error.  Later development can handle them better,
+as outlined [later](#edgeplacement-objects).
+
+### Denaturing/renaturing
+
+We could start by not doing this.  For some resources, the effect of
+leaving these resources natured in the center is only to add
+authorizations in the center that are not needed and are undesired in
+a well-secured environment but tolerable in early demonstrations ---
+provided that there is not a conflict with an object of the same name
+that is positively desired in the center.  In particular, these are:
+`ClusterRole`, `ClusterRoleBinding`, `Role`, `RoleBinding`,
+`ServiceAccount`.
+
+For some kinds of object, lack of denaturing/renaturing means that
+edge-mc will simply not be able to support workloads that contain such
+objects.  These are: `MutatingWebhookConfiguration`,
+`ValidatingWebhookConfiguration`, `LimitRange`, `ResourceQuota`.
+
```

Comment:
Are these objects currently affecting the behavior of a kcp workspace? Does kcp core has controllers watching and reconciling these objects?

Reply:
Doing `kubectl api-resources` on a kcp workspace will list those, so I expect that the kcp server is giving them interpretations.